libs/register/src/lib/register/register.component.html

```diff
@@ -0,0 +1,294 @@
+                <div
+                  class="flex text-error-light dark:text-error-dark"
+                  role=" alert"
+                >
+                  <div class="text-sm font-medium">
+                    Der Benutzername muss mindestens 5 Zeichen lang sein.
+                  </div>
+                </div>
+              </div>
+              <div *ngIf="f['name'].errors['maxlength']">
+                <div
+                  class="flex text-error-light dark:text-error-dark"
+                  role=" alert"
+                >
+                  <div class="text-sm font-medium">
+                    Der Benutzername darf maximal 30 Zeichen lang sein.
+                  </div>
+                </div>
+              </div>
+            </div>
+          </section>
+          <section>
+            <label
+              for=" email"
+              class="mt-5 block text-sm font-medium leading-6 text-on-primary-container-light dark:text-on-primary-container-dark"
+            >
+              E-Mail</label
+            >
+            <div
+              class="mb-4 flex rounded-md shadow-sm ring-1 ring-inset ring-outline-variant-light dark:ring-outline-dark sm:max-w-md"
+            >
+              <input
+                type="email"
+                name="email"
+                id="email"
+                autocomplete="email"
+                required
+                formControlName="email"
+                class="block flex-1 rounded-md border-0 bg-transparent py-1.5 pl-1 text-on-secondary-container-light placeholder:text-on-surface-variant-light focus:ring-1 dark:text-on-secondary-container-dark placeholder:dark:text-on-surface-variant-dark sm:text-sm sm:leading-6"
+                placeholder="z. B. manfred@partytime.com"
+              />
+            </div>
+            <div
+              *ngIf="
+                f['email'].errors && (f['email'].dirty || f['email'].touched)
+              "
+            >
+              <div *ngIf="f['email'].errors['required']">
+                <div
+                  class="flex text-error-light dark:text-error-dark"
+                  role=" alert"
+                >
+                  <div class="text-sm font-medium">
+                    Bitte gib deine E-Mail Adresse ein damit wir verifzieren
+                    können dass du eine echte Person bist und du zur Not dein
+                    Passwort zurück setzen kannst.
+                  </div>
+                </div>
+              </div>
+              <div *ngIf="f['email'].errors['email']">
+                <div
+                  class="flex text-error-light dark:text-error-dark"
+                  role=" alert"
+                >
+                  <div class="text-sm font-medium">
+                    Bitte gib eine Valide E-Mail Adresse ein damit wir
+                    verifzieren können dass du eine echte Person bist. Es wird
+                    eine Verfizierung an diese Adresse gesendet.
```

Comment:
```suggestion
                    verifizieren können dass du eine echte Person bist. Es wird
                    eine Verifizierung an diese Adresse gesendet.
```

@JulianTurner Auch in Cypress anpassen